metabase/metabase
# Reveal new sample db tables by default

Epic https://github.com/metabase/metabase/issues/24807

How to test:
- Go to Admin -> Databases -> Sample and trigger manual sync. New sample tables should become visible.
- Re-install Metabase and complete setup. New sample table should be visible immediately. 
- Run a cypress test. New tables should still be hidden.

<img width="823" alt="Screenshot 2022-11-15 at 15 56 50" src="https://user-images.githubusercontent.com/8542534/201937867-575aa3b7-8c49-40b4-a504-5a06c3d7b17d.png">
